homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -39,44 +43,123 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_DESTINATION): cv.string,
-        vol.Required(CONF_START): cv.string,
+        vol.Optional(CONF_DESTINATION): cv.string,
+        vol.Optional(CONF_START): cv.string,
+        vol.Optional(CONF_STATIONBOARD): vol.All(cv.ensure_list, [cv.string]),
+        vol.Optional(CONF_LIMIT): cv.positive_int,
```

Comment:
We should work with `vol.Inclusive`/`vol.Exclusive` here. Like `CONF_START` and `CONF_DESTINATION` together.  

Perhaps it could make sense to combine `CONF_START` and `CONF_STATIONBOARD` and if the `CONF_DESTINATION` is not given show the stationboard.

This is a breaking change but that's inevitable.